ecl/hql/hqlgram2.cpp

```diff
@@ -6327,6 +6337,7 @@ IHqlExpression *HqlGram::bindParameters(const attribute & errpos, IHqlExpression
                 if (requireLateBind(function, actuals))
                 {
                     IHqlExpression * ret = NULL;
+                    const bool expandCallsWhenBound = true;
```

Comment:
@ghalliday Is this local expandCallsWhenBound supposed to be set here?  If so, the next IF section (lines 6330-6340) wil not be executed.